src/infra/alarm/template.yaml

```diff
@@ -563,10 +563,11 @@ Resources:
       ActionsEnabled: true
       AlarmActions:
         - !Ref AlertingSNSHighAlertNotificationTopicARN
-      AlarmDescription: "Publishing to TxMA failed once every minute for 10 minutes"
+      AlarmDescription: "Publishing to TxMA failed once every minute for two consecutive periods over a span of 10 minutes"
```

Comment:
I don't believe the periods in which have a data point need to be consecutive. Can be any of the one minute evaluation periods over a span of 10 minutes. So may be worth removing `consecutive`.

Reply:
ohhh okay yeah that makes sense will remove